src/dapla_metadata/variable_definitions/variable_definition.py

```diff
@@ -21,6 +21,13 @@ class VariableDefinition(CompleteResponse):
         CompleteResponse: The Pydantic model superclass, representing a Variable Definition.
     """
 
+    @staticmethod
+    def from_complete_response(
+        complete_response: CompleteResponse,
+    ) -> "VariableDefinition":
```

Comment:
Why is the type in quotation marks?

Reply:
It's needed when referring to a type which hasn't yet been defined: https://peps.python.org/pep-0484/#forward-references